english/self-control/self-control.md

```diff
@@ -0,0 +1,3 @@
+Now you have everything to pass the marshmallow test. would you eat the marshmallow walls now 
+or wait when the marshmallow room would get twice bigger?  
+[Click here](http://www.amazon.com/The-Marshmallow-Test-Mastering-Self-Control/dp/0316230871) to master your self-control.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.